python/cudf_polars/tests/test_executors.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import pytest
+
+import polars as pl
+
+from cudf_polars.testing.asserts import assert_gpu_result_equal
+
+
+@pytest.mark.parametrize("executor", [None, "cudf", "dask-experimental"])
+def test_executor_basics(executor):
+    """Test basics of each executor."""
```

Comment:
*note*: Unless you want these docstrings, it should be the case that you don't need them to pacify ruff, because we have:

```
[tool.ruff.lint.per-file-ignores]
"**/tests/**/*.py" = ["D"]
```
In pyproject.toml

Reply:
Removed in https://github.com/rapidsai/cudf/pull/17262/commits/ac4d2da00dbfb2a4afe3dd6ba87f45b4618a6f68 .